### PR TITLE
Add TrackrParser tests

### LINKED ISSUE #195: Add TrackrParser tests for all Commands

### PR DESCRIPTION
**Pull request by seantaysl**
I couldn't get 3 of them to pass so I commented them out.
1. deleteTG
2. editTG
3. editStudent

### CODE REVIEW

File: src/test/java/seedu/address/testutil/TutorialGroupUtil.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import seedu.address.logic.commands.AddTutorialGroupCommand;
+import seedu.address.model.tutorialgroup.TutorialGroup;
+
+public class TutorialGroupUtil {
+    /**
+     * Returns an add command string for adding the {@code TutorialGroup}.
+     */
+    public static String getAddTutorialGroupCommand(TutorialGroup tutorialGroup) {
+        String correctArgument = " tg/B014 day/MON start/15:00 end/17:00";
+        return AddTutorialGroupCommand.COMMAND_WORD + " " + correctArgument;
```
Comment (EthanTheGoondu) [inline src/test/java/seedu/address/testutil/TutorialGroupUtil.java:12]: I think you meant to exclude one of the spaces, if not you will get a double space after the COMMAND_WORD.

File: src/test/java/seedu/address/logic/parser/TrackrParserTest.java [hunk 2]
```diff
@@ -13,58 +13,122 @@
...
+        DeleteStudentCommand command = (DeleteStudentCommand) parser.parseCommand(
+                DeleteStudentCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased());
+        assertEquals(new DeleteStudentCommand(INDEX_FIRST_PERSON), command);
+    }
+
+    @Test
+    public void parseCommand_deleteAttendance() throws Exception {
+        DeleteAttendanceCommand command = (DeleteAttendanceCommand) parser.parseCommand(
+                DeleteAttendanceCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased()
+                + " " + "week/2");
```
Comment (EthanTheGoondu) [inline src/test/java/seedu/address/logic/parser/TrackrParserTest.java]: You can put the " " with the "week/2" as one String.

File: src/test/java/seedu/address/logic/parser/TrackrParserTest.java [hunk 3]
```diff
@@ -76,26 +140,47 @@ public void parseCommand_deleteModule() throws Exception {
...
+    //                EditStudentCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased()
+    //                        + " " + StudentUtil.getEditStudentDescriptorDetails(descriptor));
+    //        assertEquals(new EditStudentCommand(INDEX_FIRST_PERSON, descriptor), command);
+    //    }
+
+    @Test
+    public void parseCommand_editParticipation() throws Exception {
+        EditParticipationCommand command = (EditParticipationCommand) parser.parseCommand(
+                EditParticipationCommand.COMMAND_WORD + " "
+                + INDEX_FIRST_PERSON.getOneBased() + " " + "score/2");
```
Comment (EthanTheGoondu) [inline src/test/java/seedu/address/logic/parser/TrackrParserTest.java:178]: You can put the " " with the "week/2" as one String.

File: src/test/java/seedu/address/testutil/AttendanceUtil.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import seedu.address.logic.commands.AddAttendanceCommand;
+import seedu.address.model.student.Attendance;
+
+public class AttendanceUtil {
+    /**
+     * Returns an add command string for adding the {@code Attendance}.
+     */
+    public static String getAddAttendanceCommand(Attendance attendance) {
+        String correctArgument = " 1 week/2";
+        return AddAttendanceCommand.COMMAND_WORD + " " + correctArgument;
```
Comment (EthanTheGoondu) [inline src/test/java/seedu/address/testutil/AttendanceUtil.java:12]: I think you meant to exclude one of the spaces, if not you will get a double space after the COMMAND_WORD.

**Review by EthanTheGoondu — COMMENTED** [review]
There seems to be a few instances of extra concatenation of strings with spaces, or double spaces, not sure if those double spaces are intentional. Otherwise the rest look good, maybe include the //todo at the commented test cases!

File: src/test/java/seedu/address/logic/parser/TrackrParserTest.java (hunk 2 above)
Comment (seantaysl) [reply src/test/java/seedu/address/logic/parser/TrackrParserTest.java]: Thanks for spotting this. Have rectified it and the rest below.

**Comment by EthanTheGoondu** [comment]
LGTM! Nice to see the increase in coverage!

**Pull request merged**